frontend/react/README.md

```diff
@@ -1,6 +1,9 @@
 ## Getting Started
+Prerequisites: 
```

Comment:
Similar to the above, having a `Prerequisites` and `Steps` subheading(h2) would be helpful.